presto-main/src/test/java/com/facebook/presto/sql/gen/TestExpressionCompiler.java

```diff
@@ -129,6 +131,13 @@
     private FunctionAssertions functionAssertions;
     private List<ListenableFuture<Void>> futures;
 
+    private static <T> T[] concat(T[] first, T[] second)
```

Comment:
Changed to Guava's `concat`. Thanks for pointing this out!
Applied in commit 87c753a99083011c29debe1d8624d27c824afe9f.